datastore/google/cloud/datastore/entity.py

```diff
@@ -183,6 +183,19 @@ def kind(self):
         if self.key:
             return self.key.kind
 
+    @property
+    def id(self):
+        """Get the id of the current entity.
+
+        .. note::
+          This relies entirely on the :class:`gcloud.datastore.key.Key`
+          set on the entity.  That means that we're not storing the id
+          of the entity at all, just the properties and a pointer to a
+          Key which knows its Id.
+        """
+        if self.key is not None:
```

Comment:
From [PEP8][1]

> Be consistent in return statements. Either all return statements in a function should return an expression, or none of them should.

So this should be:

```python
if self.key is None:
    return None
else:
    return self.key.id
```

[1]: https://www.python.org/dev/peps/pep-0008/#programming-recommendations